### PR TITLE
Release version 1.8.0

### PR DESCRIPTION
**Pull request by SpangleLabs**
This updates the changelog with #114 and bumps the version to 1.8.0, as this is a minor functional change

I'm not sure if the changelog should also contain #111 and #113, which were both landed since the last release?

### CODE REVIEW

**Review by goddenrich — APPROVED** [review]

**Comment by goddenrich** [comment]
> This updates the changelog with #114 and bumps the version to 1.8.0, as this is a minor functional change
> 
> I'm not sure if the changelog should also contain #111 and #113, which were both landed since the last release?

#111 is not a functional change it is just a refactor so doesn't need included. #113 should be added

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Version 1.8.0
+-------------
+    * Puku sync will now add `go_replace_directive` labels to any go_repo targets with a replace directive in the go.mod file (#114)
```
Comment (goddenrich) [inline ChangeLog:3]: #113 is missing

**Review by goddenrich — CHANGES_REQUESTED** [review]

File: ChangeLog (hunk 1 above)
Comment (SpangleLabs) [reply ChangeLog:3]: Added that now

**Comment by SpangleLabs** [comment]
Okay, I've added reference to #113 in the changelog now

**Review by goddenrich — APPROVED** [review]

**Pull request merged**